src/schema/schema_util.c

```diff
@@ -62,6 +62,48 @@ __wt_schema_get_source(WT_SESSION_IMPL *session, const char *name)
 	return (NULL);
 }
 
+/*
+ * __wt_schema_internal_session --
+ *	Create and return an internal schema session if necessary.
+ */
+int
+__wt_schema_internal_session(
+    WT_SESSION_IMPL *session, WT_SESSION_IMPL **int_sessionp)
+{
+	/*
+	 * Open an internal session if a transaction is running so that the
+	 * schema operations are not logged and buffered with any log records
+	 * in the transaction.  The new session inherits its flags from the
+	 * original.
+	 */
+	*int_sessionp = session;
+	if (F_ISSET(&session->txn, WT_TXN_RUNNING)) {
+		/* We should not have a schema txn running now. */
+		WT_ASSERT(session, !F_ISSET(session, WT_SESSION_SCHEMA_TXN));
+		WT_RET(__wt_open_internal_session(S2C(session), "schema",
+		    true, session->flags, int_sessionp));
+	}
+	return (0);
+}
+
+/*
+ * __wt_schema_session_release --
+ *	Release an internal schema session if needed.
+ */
+int
+__wt_schema_session_release(
+    WT_SESSION_IMPL *session, WT_SESSION_IMPL *int_session)
+{
+	WT_SESSION *wt_session;
+
+	if (session != int_session) {
+		wt_session = &int_session->iface;
+		WT_RET(wt_session->close(wt_session, NULL));
+	}
```

Comment:
Variables don't have to be used in every block, so the `WT_UNUSED` that was here shouldn't be necessary.